pom.xml

```diff
@@ -815,5 +815,18 @@
 				</dependency>
 			</dependencies>
 		</profile>
+		<profile>
+			<id>rabbitmq</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
+				<property>
+					<name>opensrp.queuing.service</name>
+					<value>rabbitmq</value>
+				</property>
+			</activation>
+			<properties>
+				<opensrp.queuing.service.profile>rabbitmq</opensrp.queuing.service.profile>
```

Comment:
add <opensrp.queuing.service.profile>default</opensrp.queuing.service.profile> on the default profile i.e postgres

Reply:
Added.